ansible-deploy.sh

```diff
@@ -0,0 +1,232 @@
+#!/bin/bash
+# Ansible-based deployment wrapper for Lean Quickstart
+# Provides similar interface to spin-node.sh but uses Ansible
+
+set -e
+
+SCRIPT_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+ANSIBLE_DIR="$SCRIPT_DIR/ansible"
+
+# Default values
+NODE_NAMES=""
+GENERATE_GENESIS=""
+CLEAN_DATA=""
+NETWORK_DIR=""
+VALIDATOR_CONFIG=""
+DEPLOYMENT_MODE="docker"
+EXTRA_VARS=""
+
+# Color codes for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+NC='\033[0m' # No Color
+
+# Help function
+show_help() {
+    cat << EOF
+Usage: $0 [OPTIONS]
+
+Ansible-based deployment for Lean Quickstart nodes.
+
+Options:
+  --node NODES              Node(s) to deploy:
+                              - 'all' for all nodes
```

Comment:
we just want it per node which the script will call node by node, the reason to not do it multiple nodes is to reduce the tech debt here